operatorcourier/cli.py

```diff
@@ -39,9 +39,11 @@ def parse(self):
     def verify(self):
         parser = argparse.ArgumentParser(description='Build and verify an operator bundle to test')
         parser.add_argument('source_dir', help='Path of your directory of yaml files to bundle. Either set this or use the files argument for bundle data.')
-        
-        args, leftovers = parser.parse_known_args(sys.argv[2:])        
-        api.build_and_verify(source_dir=args.source_dir)
+        parser.add_argument('--ui_validate_io', help='Validate bundle for operatorhub.io UI', action='store_true')
+        parser.add_argument('--ui_validate_ocp', help='Validate bundle for OCP Operator Hub UI', action='store_true')
```

Comment:
I believe this should be `OperatorHub`, since there is a pending PR in console changing the name to this officially: https://github.com/openshift/console/pull/1192